src/main/java/homeworks/homework_4/Exersise_7.java

```diff
@@ -0,0 +1,17 @@
+package homeworks.homework_4;
+
+public class Exersise_7 {
+
+    public static void main(String[] args) {
+
+        int[] array = {1, 2, 3, 4, 5};
+        int middle = 0;
```

Comment:
logic implemented not correctly.

**_Find in the array those elements whose value is less than the arithmetic mean taken from all elements of the array._**

1) find arithmetic me, for example for array  {10,2,6,8} arithmetic mean = 10+2+6+8 / 4(array.length) = 6.5
2) chek all elements in for loop, as a result you should get "2" and "6"